pxf/pxf-ignite/src/main/java/org/apache/hawq/pxf/plugins/ignite/ByteUtil.java

```diff
@@ -0,0 +1,85 @@
+package org.apache.hawq.pxf.plugins.ignite;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+import org.apache.commons.lang.ArrayUtils;
+
+/**
+ * A tool class, used to deal with byte array merging, split and other methods.
+ * IgnitePartitionFragmenter requires this class in order to work.
+ */
+public class ByteUtil {
+
+    public static byte[] mergeBytes(byte[] b1, byte[] b2) {

```

Comment:
Why not use ArrayUtils.addAll directly?

Reply:
This piece of code comes from PXF JDBC plugin; for sure, `ArrayUtils.addAll()` can be used directly; this also removes the only dependence of `ByteUtil.java`. As this package is used to process byte arrays in `IgnitePartitionFragmenter` anyway, an extra dependence may be imported there without any harm, I suppose.